Bulk broken link updates (#926)
MicrosoftDocs/biztalk-docs: biztalk/adapters-and-accelerators/adapter-sap/step-2-create-an-application-definition-file-for-the-sap-artifacts.md

## Prerequisites

> -   Be sure that you have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK.

Reviewer: Consistency edit:

```suggestion
-   You must have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK.
```
